### PR TITLE
CAMEL-19906 - Camel-Jbang: No need to have specific application prope…

### PR DESCRIPTION
**Pull request by oscerd**
…rties exports for Secret Refresh features

# Description

# Target

- [x] I checked that the commit is targeting the correct branch (note that Camel 3 uses `camel-3.x`, whereas Camel 4 uses the `main` branch)

# Tracking
- [ ] If this is a large change, bug fix, or code improvement, I checked there is a [JIRA issue](https://issues.apache.org/jira/browse/CAMEL) filed for the change (usually before you start working on it).

# Apache Camel coding standards and style

- [x] I checked that each commit in the pull request has a meaningful subject line and body.

- [x] I have run `mvn clean install -DskipTests` locally and I have committed all auto-generated changes

### CODE REVIEW

File: dsl/camel-jbang/camel-jbang-core/src/main/java/org/apache/camel/dsl/jbang/core/commands/ExportBaseCommand.java [hunk 1]
```diff
@@ -524,7 +524,6 @@ protected void copySettingsAndProfile(
             RuntimeUtil.loadProperties(prop2, profile);
         }
         prop2.putAll(prop);
-        prepareApplicationProperties(prop2);
```
Comment (davsclaus) [inline dsl/camel-jbang/camel-jbang-core/src/main/java/org/apache/camel/dsl/jbang/core/commands/ExportBaseCommand.java:527]: This call still needs to be there

Comment (oscerd) [reply dsl/camel-jbang/camel-jbang-core/src/main/java/org/apache/camel/dsl/jbang/core/commands/ExportBaseCommand.java:527]: It's used just here: https://github.com/apache/camel/pull/11528/files

For adding properties, which will be already exported by default from application.properties.. I introduced the method myself and I don't think it's needed. The prop2.putAll will do all the needed.

Comment (oscerd) [reply dsl/camel-jbang/camel-jbang-core/src/main/java/org/apache/camel/dsl/jbang/core/commands/ExportBaseCommand.java:527]: I'll check again. I'm missing something I guess.

Comment (davsclaus) [reply dsl/camel-jbang/camel-jbang-core/src/main/java/org/apache/camel/dsl/jbang/core/commands/ExportBaseCommand.java:527]: From the code then ExportQuarkus needs to have this method called as it does something special

Comment (oscerd) [reply dsl/camel-jbang/camel-jbang-core/src/main/java/org/apache/camel/dsl/jbang/core/commands/ExportBaseCommand.java]: It should be fine now.

**Review by davsclaus — APPROVED** [review]

**Pull request merged**